src/shared-components/toggle/style.js

```diff
@@ -0,0 +1,42 @@
+import styled from '@emotion/styled';
+
+import { COLORS, SPACING } from '../../constants';
+
+export const Container = styled.div`
+  position: relative;
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding: ${SPACING.small} 0;
+
+  & > input {
+    display: none;
+  }
+`;
+
+export const Label = styled.span`
+  color: ${COLORS.primaryTint1};
+  margin: 0;
+  font-size: ${SPACING.small};
+  line-height: ${SPACING.base};
+  text-align: left;
+  cursor: pointer;
+  user-select: none;
+  margin-right: ${SPACING.xsmall};
+`;
+
+export const trackStyle = {
```

Comment:
Can we make the track background color `purple15`? That's on me. It wasn't included in the design mocks but I think it will be better. It also matches our disabled button styling.

Reply:
done